jdaviz/configs/imviz/tests/test_simple_aper_phot.py

```diff
@@ -489,3 +490,26 @@ def test_curve_of_growth(with_unit):
     with pytest.raises(TypeError, match='Unsupported aperture'):
         _curve_of_growth(data, cen, EllipticalAnnulus(cen, 3, 8, 5), 100,
                          pixarea_fac=pixarea_fac)
+
+
+def test_cubeviz_batch(cubeviz_helper, spectrum1d_cube_fluxunit_jy_per_steradian):
+    cubeviz_helper.load_data(spectrum1d_cube_fluxunit_jy_per_steradian, data_label='test')
+    subset_plugin = cubeviz_helper.plugins['Subset Tools']._obj
+    phot_plugin = cubeviz_helper.plugins['Aperture Photometry']._obj
+    uc_plugin = cubeviz_helper.plugins['Unit Conversion']
+
+    fv = cubeviz_helper.app.get_viewer('flux-viewer')
+    fv.apply_roi(CircularROI(xc=5, yc=5, radius=2))
+
+    subset_plugin.subset_selected = 'Create New'
+    fv.apply_roi(CircularROI(xc=3, yc=3, radius=2))
+
+    phot_plugin.dataset_selected = 'test[FLUX]'
+    phot_plugin.multiselect = True
+    phot_plugin.aperture.selected = ['Subset 1', 'Subset 2']
+
+    uc_plugin.flux_unit = 'MJy'
+
+    phot_plugin.calculate_batch_photometry()
+
+    assert len(phot_plugin.table) == 2

```

Comment:
Should we also at least check that the sums make sense?

Reply:
I was lazily assuming that the existing tests already covered that, and we just needed to make sure that the existing code multiple times here. I could add more checks though.

Reply:
Lines are probably covered but I am unsure about how many use cases we're actually covering here and there. My take is that more tests do not hurt but it is up to you whether you want to add more checks or not.